hamocc/beleg_vars.F90

```diff
@@ -96,23 +99,20 @@ SUBROUTINE BELEG_VARS(kpaufr,kpie,kpje,kpke,kbnd,pddpo,prho,omask,      &
 
       ! local variables
       INTEGER :: i,j,k,l
-#ifdef cisonew
+      ! cisonew
       REAL :: rco213,rco214,beta13,beta14
-#endif
-#ifdef AGG
+      ! AGG
       REAL :: snow
-#endif
 
-
-#ifdef FB_BGC_OCE
-      DO k=1,kpke
-      DO j=1,kpje
-      DO i=1,kpie
-        abs_oce(i,j,k)=1.
-      ENDDO
-      ENDDO
-      ENDDO
-#endif
+      if (use_FB_BGC_OCE) then
+         DO k=1,kpke
+            DO j=1,kpje
+               DO i=1,kpie
+                  abs_oce(i,j,k)=1.
+               ENDDO
+            ENDDO
+         ENDDO
+      end if
```

Comment:
That's fine. 
I was using emacs to auto indent the if-blocks since I personally think its important to indent if segments.
And it picked up the do loops instead. I'm happy to unindent them. 
However, I would also advocate That readability is greatly improved if comments are indented as well.
That's a bigger question. That said - I've tried to keep the HAMOCC code as close as possible to its original form in this PR.